hazelcast/src/main/java/com/hazelcast/jmx/MapMBean.java

```diff
@@ -74,6 +80,8 @@ public void mapCleared(MapEvent event) {
             }
         };
         listenerId = managedObject.addEntryListener(entryListener, false);
+        StatSupplier<LocalMapStats> localMapStatsSupplier = new LocalMapStatsSupplier(managedObject);
```

Comment:
Isn't making `MapMBean` to implement `StatSupplier<LocalMapStats>` simpler? instead of creating another object?